app/services/fees/estimate_pay_in_advance_service.rb

```diff
@@ -11,9 +11,7 @@ def initialize(organization:, params:)
     end
 
     def call
-      Events::ValidateCreationService.call(
-        organization:, params:, customer:, subscriptions:, result:, send_webhook: false,
-      )
+      Events::ValidateCreationService.call(organization:, params:, customer:, subscriptions:, result:)
```

Comment:
Is it not possible to completely remove these validations? 😬

Reply:
For the estimate part, not now as the next steps of the logic relies on valid params.
Maybe in the future, but I would also say that it feels ok to have real validations on this route. We could discuss about it.